core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -423,6 +434,8 @@ public DefaultFileSystemMaster(BlockMaster blockMaster, CoreMasterContext master
     mUfsSyncPathCache = new UfsSyncPathCache();
     mSyncManager = new ActiveSyncManager(mMountTable, this);
     mTimeSeriesStore = new TimeSeriesStore();
+    mJournaledComponents =
+        Arrays.asList(mInodeTree, mDirectoryIdGenerator, mSyncManager, mMountTable, mUfsManager);
```

Comment:
does order not matter? Does sync manager depend on mount table?

Reply:
switched to match the original order just to be safe